Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
> Adding k8s.io/metrics as a vendored package was not a good idea, leading to circular repo dependencies: it depends on k8s.io/apimachinery which is in staging/ her

Well, it was desired not to have the type definitions be part of the main repository, but we need to consume apimachinery, etc as part of the API machinery for registering the API types, having autogenerated clients, etc, so I'm not sure what the solution is there (unless we want to make k8s.io/metrics a staging repo...).